### PR TITLE
Keep external db config for both Mongo and Postgres in Satellite 6.9

### PR DESCRIPTION
**Pull request by Lennonka**
This is the follow-up in addition to [#784](https://github.com/theforeman/foreman-documentation/pull/784)

Do not cherry-pick.

### CODE REVIEW

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc [hunk 1]
```diff
@@ -20,7 +20,23 @@ Use the `{foreman-installer}` command to configure {Project} to connect to exter
   --katello-candlepin-db-host _postgres.example.com_ \
   --katello-candlepin-db-name candlepin \
   --katello-candlepin-db-password _Candlepin_Password_ \
-  --katello-candlepin-manage-db false \
+  --katello-candlepin-manage-db false
+----
++
+For a PostgreSQL database, add the following arguments:
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:26]: Thank you for clarification! I didn't have the context.
So is it okay if I just put all the arguments in the single command without separation?

[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc]: I went ahead and did that.

[the remark this answers is not in the document]
Comment (ekohl) [reply guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc]: Yes, that is indeed what I was suggesting.

**Comment by ekohl** [comment]
> [x] Foreman 2.4

Generally speak we always submit a patch to the newest version first and then cherry pick it to older versions.

**Comment by ryandeussing** [comment]
@Lennonka see above ☝️

**Comment by ekohl** [comment]
> @ekohl in this case, as I understand it, the changes are already present in master and 2.5 and need to be merged to 2.3 (Satellite 6.9)

That's not what I meant. Note that the cherry pick mentioned 2.4 so then I'd expect to start at the newest version, get that merged and then pick it to 2.3. However, if all goes well 2.4 will be EOL in a month or so (and upstream 2.3 is already EOL for a while) so it doesn't matter that much.

**Comment by Lennonka** [comment]
Ryan is correct, this PR is actually meant only for 2.3 (Sat6.9) as it is based. It should not be cherrypicked to other versions of Sat, but about F2.4 I wasn't sure, that's up to you to decide.

**Comment by Lennonka** [comment]
@ekohl Ryan is correct, this PR is actually meant only for 2.3 (Sat6.9) as it is based (see the PR header). It's okay just to be merged and it doesn't need to be cherry-picked as I didn't know earlier that we should skip even versions of Foreman.

**Comment by Lennonka** [comment]
If it makes sense to fix EOL docs, of course.

**Review by ekohl — APPROVED** [review]
I'll just merge it now without cherry picking.

**Pull request merged**